### DISCUSSION TITLE
Setting up Atom - trouble with rtags

### DISCUSSION THREAD

**Discussion by swonner** [idaholab/moose#17585] [Q&A Getting Started] [answered]
I am a new user, and am working my way through tutorial 1. I have a windows system (WSL1), and am using Ubuntu. I am currently following the steps to set up Setup Atom Editor for MOOSE. However, I seem to be having some trouble setting up the source navigation with rtags. I keep getting the error:  "Could NOT find llvm-config executable and LIBCLANG_CXXFLAGS is not set". How should I fix this problem? 

 **Update**:  I was able to resolve this issue by installing llvm with "sudo apt-get install llvm". However, now I am getting the error: " Failed to compile small clang test app. It's likely that the include file <clang-c/Index.h> could not be found! Maybe you need to install the clang development package (delete the CMakeCache.txt file before trying to run cmake again after installation)?." 

I have tried the following: "sudo apt-get install llvm-dev", " sudo apt install clang", "sudo apt-get install clang", "sudo apt-get install libclang-dev", but the error persists. Any advice on how to proceed?

**Comment by aeslaughter** [comment, marked as answer]
@dschwen
